docs/100-upgrade/migration-guides/3.8-3.9.mdx

```diff
@@ -18,17 +18,6 @@ Update all your `@front-commerce/*` dependencies to this version:
 pnpm update "@front-commerce/*@3.9.0"
 ```
 
-## Automated Migration
-
-We provide a codemod to automatically update your codebase to the latest version
-of Front-Commerce. This tool will update your code when possible and flag the
-places where you need to manually update your code (with `// TODO Codemod`
-comments).
-
-```shell
-pnpm run front-commerce migrate --transform 3.9.0
```

Comment:
Shouldn't we replace with a note to ensure they ran 3.8.1 or are up-to-date (not sure that can be verified)? For users already on 3.8.0 who didn't run 3.8.1

Reply:
updated it!